src/transports/ipc.rs

```diff
@@ -218,31 +246,65 @@ async fn run_server(unix_stream: UnixStream, messages_rx: mpsc::UnboundedReceive
     }
 }
 
+fn notify(
+    subscription_txs: &mut BTreeMap<SubscriptionId, mpsc::UnboundedSender<rpc::Value>>,
+    notification: rpc::Notification,
+) -> std::result::Result<(), ()> {
+    if let rpc::Params::Map(params) = notification.params {
```

Comment:
I feel there is a lot of overlap with `ws.rs`, but I'm okay to abstract it away in case we get the third duplex transport.

Reply:
I agree with you on that one, but we were focused on bringing the duplex support back without messing around with stuff too much haha. Is it cool to leave it like that for now?

Reply:
Yup!